h/tasks/url_migration.py

```diff
@@ -0,0 +1,14 @@
+# pylint: disable=no-member # Instance of 'Celery' has no 'request' member
+from h.celery import celery
+
+
+@celery.task
```

Comment:
Oh, I think you need a `rate_limit` here!

Reply:
What value do you suggest?